cmd/linter.go

```diff
@@ -48,13 +59,13 @@ func main() {
 		EnsureKebabCasing: *ensureKebabCasing,
 		IgnoreDotfiles:    *ignoreDotfiles,
 	}
-	err = filenameslinter.CheckDir(syntaFile, os.DirFS(pwd), dirPath, &opts)
+	err = filenameslinter.CheckDir(syntaFile, os.DirFS(parent), dirPath, &opts)
 	if err != nil {
 		extra := ""
 		if *recursive {
 			extra = "recursively "
 		}
-		log.Error("Error while "+extra+"checking directory", "err", err)
+		log.Error("error while "+extra+"checking directory", "err", err)
```

Comment:
Già che ci siamo, semplifichiamo tutto questo pezzo di codice va (la diff dovrebbe rimpiazzare tutto quello che c'è dentro all'if):
```suggestion
		log.Error("error while checking directory", "recursive", *recursive, "err", err)
```